test/acceptance/features/steps/knative_serving.py

```diff
@@ -0,0 +1,39 @@
+import re
+from command import Command
+from openshift import Openshift
+from pyshould import should_not
+
+
+class Knative_Serving():
+
+    openshift = Openshift()
+    cmd = Command()
+
+    name = ""
+    namespace = ""
+
+    def __init__(self, name="knative-serving", namespace="knative-serving"):
+        self.name = name
+        self.namespace = namespace
+        self.knative_serving_yaml_template = '''
+---
```

Comment:
I guess this separator `---` is not necessary.

Reply:
Done, Thanks

Reply:
You can resolve this conversation.